contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -3073,6 +3073,576 @@ CREATE OR REPLACE FUNCTION sys.substring(string sys.NCHAR, i INTEGER, j INTEGER)
 RETURNS sys.NVARCHAR
 AS 'babelfishpg_tsql', 'tsql_varchar_substr' LANGUAGE C IMMUTABLE PARALLEL SAFE;
 
+-- wrapper functions for TRIM
+CREATE OR REPLACE FUNCTION sys.TRIM(string sys.BPCHAR)
+RETURNS sys.VARCHAR
+AS 
+$BODY$
+BEGIN
+    RETURN PG_CATALOG.btrim(string::text);
```

Comment:
what is the return data type of PG_CATALOG.btrim(string::text)? if it is not sys.varchar then we should type cast it appropriately before returning. Applicable for all other places

Reply:
return data type of `PG_CATALOG.btrim(string::text)` is `TEXT`, although in this case result of function will automatically be cast into the function's return type. refer: [link](https://www.postgresql.org/docs/current/plpgsql-control-structures.html#:~:text=will%20automatically%20be%20cast%20into%20the%20function%27s%20return%20type)